docs/guide/eslint.md

```diff
@@ -0,0 +1,18 @@
+# Eslint
+
+Peeky directly contains an Eslint plugin.
+
+For now, it automatically declares all the Peeky globals.
+In the future it may publish some rules to enforce good testing practices. 
+
+## Usage
+
+To enable this configuration use the `extends` property in your `.eslintrc`
+config file:
+
+```json
+/* .eslintrc */
+{
+  "extends": ["plugin:@peeky/recommended"]
```

Comment:
Where `@peeky/recommended` is defined ?

Reply:
Index.js, line 3.

Reply:
Thank you, I didn't see it (i'm not used to create custom eslint rules)